package.json

```diff
@@ -5,14 +5,19 @@
   "main": "./build/main.js",
   "scripts": {
     "build": "roamjs-scripts build",
-    "dev": "roamjs-scripts dev"
+    "dev": "roamjs-scripts dev",
+    "test": "TZ=UTC jest --config jest.config.ts"
   },
   "license": "MIT",
   "devDependencies": {
+    "@types/jest": "^27.0.1",
     "@types/react": "^17.0.11",
     "@types/react-dom": "^17.0.7",
+    "jest": "^27.1.0",
     "roamjs-scripts": "^0.12.3",
-    "typescript": "^4.3.2"
+    "ts-jest": "^27.0.5",
+    "ts-node": "^10.2.1",
```

Comment:
why do we need `ts-node`?

Reply:
Configuring Jest with TS, getting sweet IDE support for that.